src/helm/proxy/clients/anthropic_client.py

```diff
@@ -14,17 +15,237 @@
     TokenizationRequestResult,
     DecodeRequest,
     DecodeRequestResult,
+    TokenizationToken,
 )
 from .client import Client, wrap_request_time, truncate_sequence
+from tokenizers import Tokenizer
+from dataclasses import asdict
+
+
+class AnthropicClient(Client):
+    """
+    Client for the Anthropic models (https://arxiv.org/abs/2204.05862).
+    They use their own tokenizer.
+    Here are a list of bugs that we deal with in this client:
```

Comment:
Could you add in the issue:

- The prompt must contains `anthropic.HUMAN_PROMPT` ('\n\nHuman:') and anthropic.AI_PROMPT ('\n\nAssistant:')